Package.swift

```diff
@@ -463,6 +462,7 @@ let package = Package(
             dependencies: [
                 .target(name: "Apodini"),
                 .target(name: "ApodiniUtils"),
+                .target(name: "ApodiniOpenAPI"),
```

Comment:
As far as I can see this is still not needed for the `ApodiniDeployBuildSupport`.